README.md

```diff
@@ -13,18 +13,36 @@ https://doi.org/10.1038/s41598-022-11210-y
 On Linux and Windows the project can be used by running the following commands to clone the repository and install the required dependencies.
 
 Either with `anaconda` or `miniconda` installed
-```
+```shell
 git clone https://github.com/dslaborg/sumo.git
 cd sumo
 conda env create --file environment.yaml
 conda activate sumo
 ```
 or using `pip`
+```shell
+git clone https://github.com/dslaborg/sumo.git
+cd sumo
+pip install -r requirements.txt
 ```
+
+`pip` with virtualenv
+```shell
+# enter the project root
 git clone https://github.com/dslaborg/sumo.git
 cd sumo
+
+# Create the virtual env with pip
+virtualenv venv --python=python3.9.10
+source venv/bin/activate
```

Comment:
```suggestion
source venv/bin/activate # Windows: .\venv\Scripts\activate
```

Your script only works for Linux.

Reply:
Hi! @LarsKaulen, 
I have divided the installation guide into separate sections for Linux/MacOS and Windows. 

Please let me know if you have any further questions or need additional assistance.

Warm Regards,
@1chooo